doc/pcre2api.3

```diff
@@ -1296,8 +1296,8 @@ documentation for more details.
 .sp
   PCRE2_CONFIG_JITTARGET
 .sp
-The \fIwhere\fP argument should point to a buffer that is at least 64 code
-units long. (The exact length required can be found by calling
+The \fIwhere\fP argument should point to a buffer that is suitable aligned and
+at least 128 code units long. (The exact length required can be found by calling
```

Comment:
I would not say anything about "at least".

Reply:
"at least" is just being copied from the original wording, and agree is weird.

the next patch on this series changes the wording so it encourages the use of a dynamically allocated buffer instead, but since that "breaks API" I am not sure if it should be included in the minimal patch needed to "fix" the original issue and that should be independent enough for backport, since I don't think this is worth a "brown paper" release.